src/components/Search/SearchAutocompleteList.tsx

```diff
@@ -403,6 +412,21 @@ function SearchAutocompleteList(
                     text: value,
                 }));
             }
+            case CONST.SEARCH.SYNTAX_FILTER_KEYS.POLICY_ID: {
+                const filteredPolicies = workspaceList
+                    .filter((workspace) => workspace.name.toLowerCase().includes(autocompleteValue.toLowerCase()) && !alreadyAutocompletedKeys.includes(workspace.name.toLowerCase()))
+                    .sort()
+                    .slice(0, 10);
+
+                return canUseLeftHandBar
```

Comment:
Not super important
is it possible to do an early return with `if (canUseLeftHandBar)` above line 416? we save a bit of time on running the filter + sort.